cpp/src/arrow/c/abi.h

```diff
@@ -65,6 +65,69 @@ struct ArrowArray {
 
 #endif  // ARROW_C_DATA_INTERFACE
 
+#ifndef ARROW_C_DEVICE_DATA_INTERFACE
+#define ARROW_C_DEVICE_DATA_INTERFACE
+
+// ArrowDeviceType is compatible with dlpack DLDeviceType for portability
```

Comment:
It would be great to have docstrings for enums, new structs, etc.

Reply:
I'll fix the comment format to be docstrings. Good point

Reply:
comment formats updated to be docstrings in the same format as found in `bridge.h`